kubernetes-examples/src/main/java/io/fabric8/kubernetes/examples/LeaderElectionExamples.java

```diff
@@ -40,7 +40,9 @@
 
 @SuppressWarnings("java:S106")
 public class LeaderElectionExamples {
-
+  private LeaderElectionExamples() {
+    throw new IllegalStateException("Utility class");
+  }
```

Comment:
This is not a utility class. Changes don't apply here.

Reply:
done